lib/Avatar/AvatarWithPopover.js

```diff
@@ -29,9 +27,10 @@ const AvatarWithPopover = props => {
       placement={props.overlayPosition}
       overlay={popover}
     >
-      <span className="avatar__wrapper">
+      <button className="button button--icon-only avatar__wrapper">
         <Avatar {...props} className="avatar--with-toggle" />
-      </span>
+        {props.caret && <Icon name="caret" className="avatar--caret" />}
```

Comment:
Is `avatar--caret` a modifier or an element?

Reply:
Good point, have fixed